addons/l10n_bo/data/tax_reports/form_200_v5_tax_report_data.xml

```diff
@@ -0,0 +1,1289 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="iva_tax_report" model="account.report">
+            <field name="name">Formulario 200v5: IVA</field>
+            <field name="root_report_id" ref="account.generic_tax_report"/>
+            <field name="country_id" ref="base.bo"/>
+            <field name="filter_fiscal_position" eval="True"/>
+            <field name="availability_condition">country</field>
+            <field name="column_ids">
+                <record id="iva_tax_report_code" model="account.report.column">
+                    <field name="name">Cód. Casilla</field>
+                    <field name="expression_label">code</field>
+                    <field name="figure_type">string</field>
+                </record>
+                <record id="iva_tax_report_balance" model="account.report.column">
+                    <field name="name">Importe</field>
+                    <field name="expression_label">balance</field>
+                    <field name="figure_type">monetary</field>
+                </record>
+            </field>
+            <field name="line_ids">
+                <record id="f200_v5_tax_report" model="account.report.line">
+                    <field name="sequence" eval="0"/>
+                    <field name="name">Form. 200v.5: Impuesto al Valor Agregado IVA</field>
+                    <field name="code">f200_v5_tax_report</field>
+                    <field name="hierarchy_level" eval="0"/>
+                    <field name="children_ids">
+                        <record id="iva_debit_determination_report" model="account.report.line">
+                            <field name="sequence" eval="100"/>
+                            <field name="name">Determinación del Débito Fiscal</field>
+                            <field name="code">iva_debit_determination_report</field>
+                            <field name="children_ids">
+                                <record id="sale_base_tax_line" model="account.report.line">
+                                    <field name="sequence" eval="101"/>
+                                    <field name="name">Ventas de bienes y/o servicios Facturados en el mercado interno, excepto ventas gravadas con Tasa Cero</field>
+                                    <field name="code">iva13</field>
+                                    <field name="hide_if_zero" eval="0"/>
```

Comment:
I think it's false by default so need to put it back, also try to use "True" or "False" instead of 0 and 1 :smile: